### PR TITLE
[BABYLON] Fixes and Updates Baylon.js to latest.

### PR DESCRIPTION
**Pull request by metacritical**
* Fixes the current CDN url and Updates to latest stable Release 3.3.0
* Adds updated externs file.
* Adds checksum.edn
* Updates Build.boot to verify-checksums.

### CODE REVIEW

File: babylon/build.boot [hunk 1]
```diff
@@ -17,11 +17,12 @@
 
 (deftask package []
   (comp
-    (download :url "http://cdn.babylonjs.com/2-2/babylon.js")
```
Comment (Deraen) [inline babylon/build.boot]: Could you try

```
(comp
  (download :url (format "https://raw.githubusercontent.com/BabylonJS/Babylon.js/v%s/dist/babylon.js" +lib-version+))
  (sift :move {#"babylon.js" "cljsjs/babylon/production/babylon.min.inc.js"})
  ...)
```

So we get versioned url. Also the file doesn't need to downloaded second time, no `.min.inc.js` file is needed as the same file can be used for both development and production builds.

Comment (metacritical) [reply babylon/build.boot]: 1. Sure, will do!
2. Yes that is redundant.

Comment (metacritical) [reply babylon/build.boot]: Rebased and force pushed to a single commit.

**Comment by Deraen** [comment]
Thanks. Note, for future packages, I recommend using `validate` task instead of `validate-checksums`: https://github.com/cljsjs/packages/commit/b053e9e114c9cbf87b7a426f1d630761265d95a6

**Pull request merged**